CHANGE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v3.6.1
+- Fixed: in case of red edge from branch to its parent, the unique history of a branch never includes commits reachable from its parent.
```

Comment:
Sometimes it happens (in case of missing/incomplete reflegs, esp. on someone else's branchs) that I toggle listing commits and see literally  **h u n d r e d s** of commits for a red-edge branch, all of which (with the exception of the few latest ones) are reachable for the parent. This doesn't make sense; let's "cut" the unique range at any commit reachable from the parent branch.